index.html

```diff
@@ -626,17 +626,16 @@ <h2>Request Headers for an LDPRv</h2>
 
           <section id="LDPRv-get-response">
             <h2>Response Headers</h2>
-            <p>
-              The response to a GET request on an <a>LDPRv</a> MUST include a <code>Link: rel="original timegate"</code>
-              header referencing itself, and at least one <code>Link: rel="timemap"</code> header referencing an
-              associated <a>LDPCv</a>. It is the presence of these headers that indicates that the resource is
-              versioned.
-            </p>
-            <p>
-              The response to a GET request on an <a>LDPRv</a> MUST include a <code>Vary: Accept-Datetime</code>
-              header, exactly as per [[!RFC7089]]
-              <a href='https://tools.ietf.org/html/rfc7089#section-2.1.2'>section 2.1.2</a>.
-            </p>
+            The response to a GET request on an <a>LDPRv</a> MUST include the following <code>Link</code> headers:
```

Comment:
I think this line should still be within \<p> but you can't include the \<ul> within \<p>